seed_services_cli/ci.py

```diff
@@ -16,9 +16,9 @@ def status(ctx):
     """
     api = get_api_client(ctx.obj.ci.api_url, ctx.obj.ci.token)
     click.echo("Getting all service status")
-    result = api.get_services()
-    if result["count"] > 0:
-        for service in result["results"]:
+    results = list(api.get_services()['results'])
+    if len(results) > 0:
```

Comment:
This could just be `if results:` if we are not using the length for something else too.

Reply:
We could possibly just remove the if statement, if it is empty it would just not loop.